msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -695,5 +695,27 @@ Define the properties your app uses to post a user activity feed.
 ```
 
 ***
->
->
+
+## defaultInstallScope
+
+**Required** - string
+
+It helps you to specify the default install scopes for the apps that you publish.
+
+| **Name** | **Type** | **Maximum size** | **Required** | **Description** |
+|----------|----------|------------------|--------------|-----------------|
+| `enum` | array of strings | 4 | ✔ | Specifies whether the scope offers an experience in the context of an individual user alone (personal), a team, a groupchat, or meetings.|
+| `description` | string | 128 characters | ✔ | The description that appears to users to indicate the purpose of this command. | 
+
+## defaultGroupCapability
+
+**Required** - object
+
+Specifies the default capability for each collaboration scope your app supports.
+
+| **Name** | **Type** | **Maximum size** | **Required** | **Description** |
+|----------|----------|------------------|--------------|-----------------|
+| `properties` | string | 128 characters | ✔ | Specifies the default install scope, that is, team, groupchat, or meetings and the capability, that is, bot, tab, or connector available with the selected scope, and its description.|
```

Comment:
Please update this section to document the supported fields i.e.  groupchat, meetings and team each can have one of the following property set: tab, bot, connector. 

Ex: 
"defaultGroupCapability": {"groupchat": "bot" , "meetings": "tab" , "team": "bot"}